scripts/globals/effects/magic_def_boost.lua

```diff
@@ -14,10 +14,19 @@ effectObject.onEffectGain = function(target, effect)
 end
 
 effectObject.onEffectTick = function(target, effect)
+    -- decay the effect by 1 every effectTick
+    local boostMDB_effect_size = effect:getPower()
+    if boostMDB_effect_size > 0 then
+        effect:setPower(boostMDB_effect_size - 1)
+        target:delMod(xi.mod.MDEF, 1)
```

Comment:
if youare setting power to power -1, is it really necesary to also delete 1 point from power?

Reply:
I'm just following the examples on, e.g. VIT down 
![image](https://user-images.githubusercontent.com/16270541/207852934-7b4d1b39-23fa-43eb-b55b-c2347804cdfb.png)

Reply:
Please, dont resolve conversations, that's my job^^

Also, I want the others to look at this concrete one, so we can know if this is the proper way to do it or if we just found an issue

Reply:
oh shit sorry I thought I was supposed to resolve when I had fixed it